src/SFA.DAS.Tools.Support.UnitTests/SearchApprovalsControllerTests.cs

```diff
@@ -10,17 +20,64 @@ namespace SFA.DAS.Tools.Support.UnitTests
     public class SearchApprovalsControllerTests
     {
         private const string StopAction = "stop";
+        private const string ResumeAction = "resume";
+        public string[] Options { get; set; }
 
         [Theory, AutoMoqData]
```

Comment:
switching to NUnit should be done on this branch as it is the base branch and then merged up to MAP-183

Reply:
also please ensure you explicitly include the NUnit package